app/controllers/identification/authorisation/CheckAuthorisationAnswersController.scala

```diff
@@ -17,32 +17,36 @@
 package controllers.identification.authorisation
 
 import com.google.inject.Inject
+import config.FrontendAppConfig
 import controllers.actions.Actions
-import models.{CheckMode, Index, MovementReferenceNumber}
+import models.{CheckMode, Index, Mode, MovementReferenceNumber}
+import navigation.AuthorisationsNavigatorProvider
 import play.api.i18n.{I18nSupport, MessagesApi}
 import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
 import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendBaseController
-import viewModels.identification.CheckAuthorisationAnswersViewModel.CheckAuthorisationAnswersViewModelProvider
+import viewModels.identification.AuthorisationAnswersViewModel.CheckAuthorisationAnswersViewModelProvider
 import views.html.identification.authorisation.CheckAuthorisationAnswersView
 
 class CheckAuthorisationAnswersController @Inject() (
   override val messagesApi: MessagesApi,
+  navigatorProvider: AuthorisationsNavigatorProvider,
   actions: Actions,
   val controllerComponents: MessagesControllerComponents,
   view: CheckAuthorisationAnswersView,
   viewModelProvider: CheckAuthorisationAnswersViewModelProvider
-) extends FrontendBaseController
+)(implicit config: FrontendAppConfig)
```

Comment:
Do we still need to be passing the FrontEndAppConfig in all these controllers etc?